extensions/cornerstone/src/commandsModule.ts

```diff
@@ -15,21 +15,23 @@ import { ServicesManager } from '@ohif/core';
 import CornerstoneViewportDownloadForm from './utils/CornerstoneViewportDownloadForm';
 import callInputDialog from './utils/callInputDialog';
 import { setColormap } from './utils/colormap/transferFunctionHelpers';
-import toggleMPRHangingProtocol from './utils/mpr/toggleMPRHangingProtocol';
 import toggleStackImageSync from './utils/stackSync/toggleStackImageSync';
 import getActiveViewportEnabledElement from './utils/getActiveViewportEnabledElement';
 
-const commandsModule = ({ servicesManager }) => {
+const commandsModule = ({
+  servicesManager,
+}: {
+  servicesManager: ServicesManager;
+}): React.FunctionComponent => {
```

Comment:
I konw `React.FunctionComponent` is wrong, but do we have react stuff below? then ok